src/FakeItEasy.dotnet/FakeItEasy.dotnet.nuget.targets

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8" standalone="no"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(NuGetPackageRoot)' == ''">
+    <NuGetPackageRoot>$(UserProfile)\.nuget\packages\</NuGetPackageRoot>
+  </PropertyGroup>
+  <ImportGroup>
+    <Import Project="$(NuGetPackageRoot)\StyleCop.MSBuild\4.7.54\build\StyleCop.MSBuild.Targets" Condition="Exists('$(NuGetPackageRoot)\StyleCop.MSBuild\4.7.54\build\StyleCop.MSBuild.Targets')" />
+  </ImportGroup>
+</Project>
```

Comment:
Interesting that NuGet does this when the project has a `project.json`, even though there is also a `csproj` where it could have been added.

Also interesting that we initially had these files ignored in the initial .NET PR (fixed in #816). I don't see how we can ignore these files.